src/js/core/widget/core/Appbar.js

```diff
@@ -507,15 +508,17 @@
 					totalHeight = min(totalHeight, nominalHeights.expanded);
```

Comment:
It's not something you've modified, but I think it's good to use capital letters.